core/services/workflows/v2/engine.go

```diff
@@ -384,6 +387,9 @@ func (e *Engine) startExecution(ctx context.Context, wrappedTriggerEvent enqueue
 		return
 	}
 
+	e.metrics.UpdateTotalWorkflowsGauge(ctx, executingWorkflows.Add(1))
```

Comment:
We recently discussed that this gauge is not super useful as it only shows executions at a single point in time. But let's keep it for now, until we improve.